runtime/src/test/scala/com/ing/baker/runtime/core/BakerInquireSpec.scala

```diff
@@ -0,0 +1,71 @@
+package com.ing.baker.runtime.core
+
+import akka.persistence.inmemory.extension.{InMemoryJournalStorage, StorageExtension}
+import akka.testkit.TestProbe
+import com.ing.baker._
+import com.ing.baker.compiler.RecipeCompiler
+import com.ing.baker.il.CompiledRecipe
+import com.ing.baker.recipe.TestRecipe.getRecipe
+import org.slf4j.LoggerFactory
+
+import scala.language.postfixOps
+
+class BakerInquireSpec extends BakerRuntimeTestBase {
+
+  override def actorSystemName = "BakerInquireSpec"
+
+  val log = LoggerFactory.getLogger(classOf[BakerInquireSpec])
+
+
+  before {
+    resetMocks
+    setupMockResponse()
+
+    // Clean inmemory-journal before each test
+    val tp = TestProbe()
+    tp.send(StorageExtension(defaultActorSystem).journalStorage, InMemoryJournalStorage.ClearJournal)
+    tp.expectMsg(akka.actor.Status.Success(""))
+  }
+
+  "Baker" should {
+
+    "return recipe if asked" in {
+      val (baker, recipeId) = setupBakerWithRecipe("returnRecipe", false)
+      val recipe: CompiledRecipe = baker.getRecipe(recipeId)
+      recipe.name shouldBe "returnRecipe"
+    }
+
+    "return all recipes if asked" in {
+      val (baker, recipeId) = setupBakerWithRecipe("returnAllRecipes", false)
+      baker.addRecipe(RecipeCompiler.compileRecipe(getRecipe("returnAllRecipes2")))
+      val recipes: Map[String, CompiledRecipe] = baker.getAllRecipes()
+      recipes.size shouldBe 2
+      recipes(recipeId).name shouldBe "returnAllRecipes"
+    }
+
+    "return health of a recipe if asked" in {
+      val (baker, recipeId) = setupBakerWithRecipe("returnHealthRecipe", false)
+      val recipeHealth: RecipeHealth = baker.getRecipeHealth(recipeId)
```

Comment:
Better to do these kind of assertions using the 'have' matchers. see example below:
```
import org.scalatest.Matchers._
case class Book(title: String, author: String, pubYear: Int)
val book = Book("A Book", "Sally", 2008)

book should have (                                 // check arbitrary properties dynamically (via reflection)
  'title ("A Book"),                               // see HavePropertyMatcherGenerator
  'author ("Sally"),
  'pubYear (2008)
)
```